rpm/patchmanager.spec

```diff
@@ -207,7 +205,8 @@ dbus-send --system --type=method_call --dest=org.freedesktop.DBus / org.freedesk
 systemctl daemon-reload
 systemctl-user daemon-reload
 
-%files test-cases
+%files %{name}-testcases
```

Comment:
Same here, see comment on `%package` above.

Reply:
… and a third time for the sub-package's `%description`.